templates/baremetal_deployment.yaml.j2

```diff
@@ -1,6 +1,8 @@
 - name: Controller
   count: {{ controller_count }}
   hostname_format: controller-%index%
+  ansible_playbooks:
```

Comment:
This change is also in #486. It will lead to merge conflicts. I think this change can be removed from this PR and rebased when the #486 is merged into the master branch. If this PR depends on #486, please mention that in the commit message.

Reply:
okay